src/styles/foundation/utilities.scss

```diff
@@ -1,62 +1,69 @@
-$default-browser-font-size: 16px;
-$base-font-size: 10px;
+$root-font-size: 16px;
```

Comment:
I would really like this to be a constant but Sass has no notion of private variables without pain, so I’m leaving it here. (The alternative is hard-coding `16px` into these calculations which I like even less.)

Reply:
You're all good! The way it's declared, it's basically a constant. People can't easily override it (compared to a variable declared using a `!default` flag).

If you wanna make it _look_ private, prefix it with a `_`, it's a very common convention in the Sass community but I don't know if it is at Shopify.

```scss
// Easy to override
$root-font-size: 16px !default;

// Hard to override (have to fiddle with imports)
$root-font-size: 16px;

// Don't override me!
$_root-font-size: 16px;
```